grumblr/app/assets/javascripts/application.js

```diff
@@ -13,4 +13,20 @@
 //= require jquery
 //= require jquery_ujs
 //= require turbolinks
+//= require underscore
+//= require backbone
+//= require handlebars.runtime
+//= require_tree ./templates
+//= require_tree ./backbone/routers
+//= require_tree ./backbone/models
+//= require_tree ./backbone/collections
+//= require_tree ./backbone/views
 //= require_tree .
+$(document).ready(loadApp);
+
+function loadApp() {
+	mainCollection = new Grumbles();
+  	mainCollectionView = new GrumbleListView({collection: mainCollection});
+	mainCollection.fetch();
```

Comment:
I think you need to pass in the object `{reset: true}` in order to fire off the reset event on fetch